src/components/Calls/CallIFrame.js

```diff
@@ -1,14 +1,18 @@
 import { h } from 'preact';
 
-import { Screen } from '../Screen';
+import { Livechat } from '../../api';
 import { createClassName } from '../helpers';
 import styles from './styles.scss';
 
+const { localStorage } = window;
+const storedState = JSON.parse(localStorage.getItem('store'));
```

Comment:
We have a dedicated store class to fetch this info so I think we should be using that
https://github.com/RocketChat/Rocket.Chat.Livechat/blob/develop/src/store/index.js